### ISSUE TITLE
Does this script currently support L20 and H20?

### ISSUE THREAD

**Issue by ychjiang** [NVIDIA/gpu-admin-tools#6]
hi，I haven't found the register description regarding L20 and H20. 
Does it currently support setting ECC on L20 and H20?

https://github.com/NVIDIA/gpu-admin-tools/blob/main/nvidia_gpu_tools.py#L1061

**Comment by ychjiang** [comment]
When I run this script on an A100 with the command : 
  python3  ./nvidia_gpu_tools.py --force-ecc-on-after-reset --gpu 0 --recover-broken-gpu 
  it takes effect. 
However, it doesn't work on L20 and H20.

**Comment by pjaroszynski** [comment]
Yes, H20 and L20 should work. Newly pushed https://github.com/NVIDIA/gpu-admin-tools/releases/tag/v2024.12.06 makes it a bit more obvious by picking up the right "H20" and "L20" names for these GPUs.

Let me know what specifically is not working and capture a log with `--log debug`.
